wiki/Developer-Documentation.md

```diff
@@ -109,205 +93,70 @@ application is as follows:
    client library will fail-over to another chunk server that has the data; this
    fail-over is transparent to the application.
 
-Compiling C++ Side
+Compiling the C++ Code
 ------------------
-Compiling the C++ side produces metaserver and chunk server binaries, client and
-admin tools, and the C++ client library. We use **`cmake`** to build the C++
-code. Here are the steps:
-
-1. Change directory to the build root (i.e. `~/code/qfs/build/release`).
-
-        $ cd ~/code/qfs && mkdir -p build/release && cd build/release
-
-1. Generate the makefiles.
-
-        $ cmake ~/code/qfs/
-
-**Note:** If you have non-default locations for your boost or java files, then
-set `BOOST_ROOT`, `BOOST_LIBRARYDIR`, and `JAVA_INCLUDE_PATH` accordingly.
-
-        $ cmake -D BOOST_ROOT=/usr/local/include/boost-1_37 \\
-                -D BOOST_LIBRARYDIR=/usr/local/lib \\
-                -D JAVA_INCLUDE_PATH=/usr/java/default/include \\
-                ~/code/qfs
-
-1. Compile the source
-
-        $ make
-
-1. Once compilation is success, install the binaries. **Note:** Although the
-   following command suggests system installation what it actually does is to
-   just copy the binaries and libraries to separate directories under the build
-   directory.
-
-        $ make install
-
-At the end of the last step, you will have the following hierarchy:
-
-| Directory                    | Notes                          |
```

Comment:
This hierarchy seems to be a useful information. I think we should keep it after updating if necessary.